source/initial-dns-seedlist-discovery/tests/README.md

```diff
@@ -3,6 +3,31 @@
 This directory contains platform-independent tests that drivers can use to prove their conformance to the Initial DNS
 Seedlist Discovery spec.
 
+### Prose Tests
+
+1. When given the following valid SRVs, parse the connection string without throwing a parse related error.
+
+   a. Only one domain level (ex: `mongodb+srv://localhost`).
+
+   b. Only two domain levels (ex: `mongodb+srv://mongo.local`).
+
+<br />
```

Comment:
Do you need `<br />`? a blank line should do the trick in markdown

Reply:
A blank line doesn't show the distinction between the tests imo since there's blank lines between them, too. The `<br />` adds another line on top of the blank line. Happy to change it if you feel its clear either way.

Reply:
Odd, let's just match formatting from other prose markdown files, I think putting html into markdown is generally undesirable and I don't see `<br />` used in other prose test files.